lib/rescuecdstep.pm

```diff
@@ -6,6 +12,11 @@ use warnings;
 
 # Base class for all RESCUECD tests
 
+=head2 test_flags
+
+Return test flag fatal => 1
```

Comment:
Something may have been wrong with this. The conversation is shown as resolved, but I cannot see the changes.

Reply:
ah, change was not there.

Reply:
it should be okay now.

Reply:
sorry, but I still can't see the changes. There is also no indication on github, that you pushed again.

https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/8533.patch should contain the changes